ui/app/components/app/connected-accounts-list/index.scss

```diff
@@ -0,0 +1,161 @@
+.connected-accounts-list {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  &__identicon {
+    margin-right: 8px;
+  }
+
+  &__account-name {
+    font-weight: bold;
+    font-size: 14px;
+    line-height: 20px;
+  }
+
+  &__primary {
+    font-style: italic;
+  }
+
+  &__account-status {
+    color: $Grey-500;
+  }
+
+  &__account-status-link {
+    &, &:hover {
+      color: $curious-blue;
+      cursor: pointer;
+    }
+  }
+
+  &__primary, &__account-status {
+    font-size: 12px;
+    line-height: 17px;
+  }
+
+  &__row {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    padding: 16px 24px;
+
+    border-top: 1px solid $geyser;
+
+    &:last-of-type {
+      border-bottom: 1px solid $geyser;
+    }
+
+    &--highlight {
+      background-color: $warning-light-yellow;
+      border: 1px solid $warning-yellow;
+
+      box-sizing: border-box;
+      padding: 16px;
+      margin-bottom: 16px;
+      width: calc(100% - 16px)
+    }
+  }
+
+  &__row-content {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+  }
+}
+
+.connected-accounts-options-button {
+  background: none;
+}
+
+.connected-accounts-options {
+  background: $white;
+  box-shadow: 0 2px 10px rgba(0, 0, 0, 0.214);
+  border-radius: 8px;
+  min-width: 200px;
+  color: #24292E;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0 16px;
+  right: 24px;
+
+  font-family: Roboto, 'sans-serif';
+  font-size: 14px;
+  font-weight: normal;
+  line-height: 20px;
+
+  &__row {
+    background: none;
+    font-family: inherit;
+    font-size: inherit;
+
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    width: 100%;
+    padding: 16px 0;
```

Comment:
Nit: The rows are shorter in the designs.

```suggestion
    padding: 14px 0;
```

It doesn't look terrible at 16px though! :thinking: 

<details>
<summary>Screenshots: </summary>

Before:

![before-popup-small](https://user-images.githubusercontent.com/2459287/81982315-0241e880-9608-11ea-8eb4-2ffc168c016b.png)

After:

![after-popup-small](https://user-images.githubusercontent.com/2459287/81982321-040bac00-9608-11ea-8caa-aba88da98849.png)

</details>